CodeSniffer/Standards/Generic/Tests/Formatting/DisallowLongArraySyntaxUnitTest.fail.inc

```diff
@@ -0,0 +1,20 @@
+<?php
```

Comment:
The current implementation of  `AbstractSniffUnitTest::testSniff` will consider test files matching `FooUnitTest.*` for a sniff called `FooSniff`.